scipy/interpolate/_bsplines.py

```diff
@@ -561,6 +589,96 @@ def integrate(self, a, b, extrapolate=None):
         integral *= sign
         return integral.reshape(ca.shape[1:])
 
+    @classmethod
+    def from_power_basis(cls, pp, bc_type='not-a-knot'):
+        r"""
+        Construct a polynomial in the B-spline basis
+        from a piecewise polynomial in the power basis.
+        
+        For now, accepts ``CubicSpline`` instances only.
+
+        Parameters
+        ----------
+        pp : CubicSpline
+            A piecewise polynomial in the power basis, as created by CubicSpline
+        bc_type : string, optional
+            Boundary condition type as in CubicSpline:
+            one of the ``not-a-knot``, ``natural``, ``clamped``, or ``periodic``
+            ``bc_type`` is needed for composition a knot vector.
```

Comment:
- Consider clarifying that these are boundary conditions *for the bspline*. 
- if it's optional (good!) please specify the default.
- the last sentence is probably not needed here. Maybe Notes are more appropriate if at all.